firedrake/interpolation.py

```diff
@@ -643,7 +653,8 @@ def _interpolate(
         self,
         *function,
         output=None,
-        transpose=False,
+        transpose=None,
+        adjoint=False,
```

Comment:
Using an `adjoint` argument unfortunately hides the `adjoint` function. I've worked around this with `expr_adjoint` but that doesn't seem ideal.